hack/operatorhub/Makefile

```diff
@@ -2,5 +2,27 @@
 # or more contributor license agreements. Licensed under the Elastic License 2.0;
 # you may not use this file except in compliance with the Elastic License 2.0.
 
+DRY_RUN ?= true
+TAG     ?= $$(cat ../../VERSION)
+
 build:
 	go build -o bin/operatorhub main.go
+
+container-push:
+	bin/operatorhub container push --dry-run=$(DRY_RUN)
+
+preflight:
+	../../.buildkite/scripts/release/redhat-preflight.sh $(TAG)
+
+container-publish:
+	bin/operatorhub container publish --dry-run="$(DRY_RUN)"
+
+generate-manifests:
+	bin/operatorhub generate-manifests --yaml-manifest=../../config/crds.yaml --yaml-manifest=../../config/operator.yaml
+
+bundle-generate:
+	bin/operatorhub bundle generate --dir="$$(pwd)"
```

Comment:
Isn't the `$$` only needed in Buildkite?

```
❯ dir="$$(pwd)"; echo $dir
70487(pwd)

❯ dir="$(pwd)"; echo $dir
/Users/mmontgomery
```

Reply:
Here we don't want the variable `$pwd` but run the shell command `$(pwd)` and `$` is a special character in make that needs to be escaped. I can use the other syntax `$(shell pwd)` which is less confusing.